specification/sql/resource-manager/readme.md

```diff
@@ -24,13 +24,111 @@ These are the global settings for the Sql API.
 title: SqlManagementClient
 description: 'The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables you to create, retrieve, update, and delete databases.'
 openapi-type: arm
-tag: package-composite-v4
+tag: package-composite-v5
 ```
 
 ## Composite packages
 
 The following packages may be composed from multiple api-versions.
 
+### Tag: package-composite-v5
+
+These settings apply only when `--tag=package-composite-v5` is specified on the command line.
+
+This section contains the "composite-v5" set of APIs, which is composed from a selection of api-versions that will remain backwards compatible with "v5" clients.
+
+APIs must only be added to this section when the API is publicly available in at least 1 production region and at least 1 generated client has been tested end-to-end.
+
+Differences in v5 (compared to v4):
+
+* Added new API for databases
+
+  * `-2020-11-01/databases.json`
+
+``` yaml $(tag) == 'package-composite-v5'
```

Comment:
@ericshape is working  another PR for V5 SDK based on 2020=08-01-preview.  is there any conflict?
https://github.com/Azure/azure-rest-api-specs/pull/12107

Reply:
Just took a look.  @ericshape uses 2020-08-01-preview edition of Databases.json while I use 2020-11-01-preview version.  Besides this I don't see any conflict.  What is the best way to proceed? I would like to update the latest version of the Databases swagger to be 2020-11-01 version.

Reply:
it depends on the merge order. 
if #12146 is merged ahead of #12107, I will update the v5 SDK. 
if #12146 is merged after #12107, @xaliciayang would replace 2020-08-01 Databases.json with the newly added 2020-11-01 one.

Reply:
> > Summary of discussion with @akning-ms
> > Currently, storageAccountType property from (found in Databases.json from 2020-08-01-preview and before) is only used by _whitelisted_ subscriptions. Customers who are not whitelisted for this feature are not able to change this value.
> > We received feedback from customers who are currently using this property to change the name to be less vague. Therefore we decided upon currentBackupStorageRedundancy/requestedBackupStorageRedundancy for the following reasons:
> > 
> > 1. the configuration is for database **backup** storage and not database storage
> > 2. since the value will not immediately change upon PUT/PATCH request, we added the requested* property to reflect that the customer has requested a new backup storage option, and it can be different from the current* property.
> 
> Thanks @xaliciayang for detail, is this whitelist controlled by AFEC in ARM? but looks AFEC not support property level whitelist. so it is supported in service itself by subscription id? Is this property show in response?

Yes this is controlled in the service, specifically by feature switch in each region or at subscription level.  Yes, both of these properties appear in the GET Database response